File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -77,375 +78,332 @@ private void InstallDependecies(string shellFilePath)
+            //Convert HTML string to PDF
+            PdfDocument document = htmlConverter.Convert("http://www.syncfusion.com");
+            //Save the document into stream
+            MemoryStream stream = new MemoryStream();
+            document.Save(stream);
+            stream.Position = 0;
+            //Close the document
+            document.Close(true);
+            //Defining the ContentType for pdf file
+            string contentType = "application/pdf";
+            //Define the file name
+            string fileName = "URL_to_PDF.pdf";
+            //Creates a FileContentResult object by using the file contents, content type, and file name
+            return File(stream, contentType, fileName);
+        }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-### Refer to the following steps to publish as Azure App Linux:
+## Refer to the following steps to publish as Azure App Linux
 
-Right-click the project and select Publish.
+1. Right-click the project and select Publish.
 ![Convert HTMLToPDF Azure NetCore Step7](htmlconversion_images/AzureNetCore7.png)
 
-Create a new profile in publish target window.
+2. Create a new profile in publish target window.
 ![Convert HTMLToPDF Azure NetCore Step8](htmlconversion_images/AzureNetCore8.png)
 
-Create App service using Azure subscription and select a hosting plan.
+3. Create App service using Azure subscription and select a hosting plan.
 ![Convert HTMLToPDF Azure NetCore Step9](htmlconversion_images/AzureNetCore9.png)
 
-HTML to PDF conversion works from basic hosting plan (B1 to P3). So, select the hosting plan as required. HTML to PDF conversion will not work if the hosting plan is Free/Shared.
+4. HTML to PDF conversion works from basic hosting plan (B1 to P3). So, select the hosting plan as required. HTML to PDF conversion will not work if the hosting plan is Free/Shared.
 ![Convert HTMLToPDF Azure NetCore Step10](htmlconversion_images/AzureNetCore10.png)
 
-After creating a profile, click the publish button.
+5. After creating a profile, click the publish button.
 ![Convert HTMLToPDF Azure NetCore Step11](htmlconversion_images/AzureNetCore11.png)
 
-Now, the published webpage will open in the browser. Click Export to PDF to convert the Syncfusion webpage to a PDF.
+6. Now, the published webpage will open in the browser. Click Export to PDF to convert the Syncfusion webpage to a PDF.
 ![Convert HTMLToPDF Azure NetCore Step11](htmlconversion_images/htmltopdfoutput.png)
 
 A complete work sample for converting an HTML to PDF in Azure App service on Linux can be downloaded from [AzureAppLinux_CoreSample.zip.](https://www.syncfusion.com/downloads/support/directtrac/general/ze/AzureAppLinux_CoreSample-2144256654)
 
 ## Azure App Service Linux with docker
 
-### Steps to convert HTML to PDF in Azure app service using the Blink with Linux docker container:
+### Steps to convert HTML to PDF in Azure app service using the Blink with Linux docker container
```

Comment:
Steps to convert HTML to PDF in Azure app service using Blink with Linux docker container

Reply:
changed the content